staging/src/k8s.io/client-go/pkg/apis/clientauthentication/v1alpha1/types.go

```diff
@@ -58,6 +58,9 @@ type ExecCredentialStatus struct {
 	ExpirationTimestamp *metav1.Time `json:"expirationTimestamp,omitempty"`
 	// Token is a bearer token used by the client for request authentication.
 	Token string `json:"token,omitempty"`
+	// PEM-encoded client TLS certificate and key.
```

Comment:
Document both fields separately so the generated documentation makes sense. Document what this key is used for (who is the client and who is the server). Our certs, cas, and keys are VERY confusing already. What may be in this certificate data? Is it a ca bundle? A single cert?

Reply:
Split comments and elaborated on cert contents